### PR TITLE
Added minimum required functions to mob stubs. Commented out more blocks

### PR DESCRIPTION
**Pull request by mwvent**
Added minimum required functions to mob stubs. Commented out some more blocks that havent been implemented yet to allow starting server

### CODE REVIEW

**Comment by mwvent** [comment]
Dont merge this yet it wont start now due to Human.php let me check it

File: src/pocketmine/block/Block.php [hunk 1]
```diff
@@ -630,11 +630,11 @@ public static function init(){
 			self::$list[self::FENCE_GATE_DARK_OAK] = FenceGateDarkOak::class;
 			self::$list[self::FENCE_GATE_ACACIA] = FenceGateAcacia::class;
 			
-			self::$list[self::SPRUCE_DOOR_BLOCK] = SpruceDoor::class;
-			self::$list[self::BIRCH_DOOR_BLOCK] = BirchDoor::class;
-			self::$list[self::JUNGLE_DOOR_BLOCK] = JungleDoor::class;
-			self::$list[self::ACACIA_DOOR_BLOCK] = AcaciaDoor::class;
-			self::$list[self::DARK_OAK_DOOR_BLOCK] = DarkOakDoor::class;
+			//self::$list[self::SPRUCE_DOOR_BLOCK] = SpruceDoor::class;
```
Comment (DRedDogPE) [inline src/pocketmine/block/Block.php]: I have added these

File: src/pocketmine/entity/Chicken.php [hunk 2]
```diff
@@ -23,5 +23,7 @@
 
 
 class Chicken extends Animal{
-
-}
\ No newline at end of file
+	public function getName() {
+		return "Chicken";
```
Comment (DRedDogPE) [inline src/pocketmine/entity/Chicken.php]: I have fixed this (not published yet)

**Comment by mwvent** [comment]
Ok no its OK I guess somebody was working on it and hasn't updated Player.php yet

**Comment by mwvent** [comment]
Cancelled - I am duplicating work!

**Pull request closed without merging**